stream-chat-android-docs/src/main/java/io/getstream/chat/docs/java/Channels.java

```diff
@@ -52,67 +63,149 @@ public void initialization() {
      */
     class CreatingChannels {
         public void createAChannel() {
-            String channelType = "messaging";
-            String channelId = "id";
+            ChannelClient channelClient = client.channel("messaging", "general");
+
             Map<String, Object> extraData = new HashMap<>();
-            client.createChannel(channelType, channelId, extraData).enqueue(result -> {
-                if (result.isSuccess()) {
-                    Channel newChannel = result.data();
-                } else {
-                    Log.e(TAG, String.format("There was an error %s", result.error()), result.error().getCause());
-                }
-            });
+            channelClient.create(extraData)
+                    .enqueue(result -> {
+                        if (result.isSuccess()) {
+                            Channel newChannel = result.data();
+                        } else {
+                            // Handle result.error()
+                        }
+                    });
         }
     }
 
     /**
      * @see <a href="https://getstream.io/chat/docs/watch_channel/?language=java">Watching A Channel</a>
      */
     class WatchingAChannel {
-        public void watchingChannel() {
+        public void watchingASingleChannel() {
+            ChannelClient channelClient = client.channel("messaging", "general");
+
             channelClient.watch().enqueue(result -> {
                 if (result.isSuccess()) {
                     Channel channel = result.data();
                 } else {
-                    Log.e(TAG, String.format("There was an error %s", result.error()), result.error().getCause());
+                    // Handle result.error()
+                }
+            });
+        }
+
+        public void watchingMultipleChannels(String currentUserId) {
+            FilterObject filter = Filters.and(
+                    Filters.eq("type", "messaging"),
+                    Filters.in("members", Arrays.asList(currentUserId))
+            );
+            int offset = 0;
+            int limit = 10;
+            QuerySort<Channel> sort = new QuerySort<Channel>().desc("last_message_at");
+            int messageLimit = 0;
+            int memberLimit = 0;
+
+            QueryChannelsRequest request = new QueryChannelsRequest(filter, offset, limit, sort, messageLimit, memberLimit)
+                    .withWatch() // Watches the channels automatically
+                    .withState();
```

Comment:
Maybe we should also explain what `withState()` does?

Reply:
Explained in the written part of the docs to some degree, but will double-check its behaviour later